controllers/api/tweetController.js

```diff
@@ -16,11 +16,14 @@ const tweetController = {
     */
     try {
       const tweets = await Tweet.findAll({
-        include: [Reply, Like]
+        include: [Reply, Like, User]
```

Comment:
exclude password for security concern.
`include: [{ model: User, as: 'following', attributes: { exclude: ['password'] } }]`

Reply:
ok